baseplate/integration/trace_headers.py

```diff
@@ -0,0 +1,122 @@
+import re
+
+
+LOWER_HEX_16 = re.compile(r"\A[0-9a-f]{16}\Z")
+LOWER_HEX_32 = re.compile(r"\A[0-9a-f]{32}\Z")
+
+LOWER_HEX_16_BYTES = re.compile(b"\A[0-9a-f]{16}\Z")
+LOWER_HEX_32_BYTES = re.compile(b"\A[0-9a-f]{32}\Z")
+
+
+class TraceHTTPHeaders(object):
+    """Helper class for trace headers in an HTTP call.
+
+    """
+    HEADER_NAMES = {
+        "trace_id": ["X-Trace", "X-B3-TraceId"],
```

Comment:
Should we use tuples rather than lists for these?  Tuples are a little smaller and immutable.

Reply:
Also, the way these are ordered will favor the old header format over the new, maybe we should put the new one first?